test/functional/ui/input_spec.lua

```diff
@@ -406,6 +406,61 @@ describe('input non-printable chars', function()
   end)
 end)
 
+describe('skip do you really want to write on force write', function()
+  after_each(function()
+    os.remove('Xtest-overwrite-forced')
+  end)
+
+  it("does not ask for confirmation when force write", function()
+    write_file("Xtest-overwrite-forced", [[foobar]])
+    local screen = Screen.new(40,4)
+    screen:set_default_attr_ids({
+      [1] = {bold = true, foreground = Screen.colors.Blue1},
+      [2] = {foreground = Screen.colors.Grey100, background = Screen.colors.Red},
+      [3] = {bold = true, foreground = Screen.colors.SeaGreen4}
+    })
+    screen:attach()
+    command("set display-=msgsep shortmess-=F")
+
+    feed_command("e Xtest-overwrite-forced")
+    screen:expect([[
+      ^foobar                                  |
+      {1:~                                       }|
+      {1:~                                       }|
+      "Xtest-overwrite-forced" [noeol] 1L, 6B |
+    ]])
+
+    -- The timestamp is in second resolution, wait two seconds to be sure.
+    screen:sleep(2000)
```

Comment:
The test could be a lot faster by manually changing the time via `utime`:

https://github.com/luvit/luv/blob/master/docs.md#uvfs_utimepath-atime-mtime-callback

    helpers.nvim_exec_lua([[vim.loop.fs_utime('Xtest-overwrite-forced', 999999999, 999999999)]])

This is worth the trouble to save 2 seconds. If all of our tests took 1 second, the test suite would take 66 hours to run.

Reply:
I did not used this exact line, but It should achieve the same result.